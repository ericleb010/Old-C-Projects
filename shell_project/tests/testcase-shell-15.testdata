c1=echo
v1=seven
${c1} ${v1}
