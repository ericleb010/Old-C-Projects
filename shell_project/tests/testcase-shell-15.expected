seven
Child (<0>) exited -- success (Status: 0)

------------------------------------
Welcome to the Assignment Shell!
At any time, you can call "help" to read the documentation.

	>>> 	>>> 	>>> 	>>> 
